examples/complete/main.tf

```diff
@@ -2,6 +2,10 @@ provider "aws" {
   region = var.region
 }
 
+locals {
+  enabled = module.this.enabled
+}
+
 data "aws_caller_identity" "current" {}
```

Comment:
I have mixed feelings about it, because I am more interested in testing the module with `enabled == false` when it has valid inputs, but of course it is also important to ensure that invalid inputs do not cause problems when `enabled = false`. 

I will go ahead and make the changes.